All unit tests are passed in CI test runner, however, CI fails due to inconsistent eslint rules between the project repo and CI. Should be fixed now. 

Output snippet
```
> pr-merge@1.0.0 test /home/travis/build/sfdye/pr-merge
> jest && standard
 PASS  __tests__/index.test.js
  app plugin test
    should resolve
      ✓ should performs an action (11ms)
    should reject
      ✓ should throw exception (8ms)
10:24:55.170Z ERROR probot: 
  Error
      at Suite.describe (/home/travis/build/sfdye/pr-merge/__tests__/index.test.js:55:27)
      at addSpecsToSuite (/home/travis/build/sfdye/pr-merge/node_modules/jest-jasmine2/build/jasmine/Env.js:334:25)
      at Env.describe (/home/travis/build/sfdye/pr-merge/node_modules/jest-jasmine2/build/jasmine/Env.js:303:7)
      at describe (/home/travis/build/sfdye/pr-merge/node_modules/jest-jasmine2/build/jasmine/jasmine_light.js:97:157)
      at Suite.Object.<anonymous>.describe (/home/travis/build/sfdye/pr-merge/__tests__/index.test.js:52:3)
      at addSpecsToSuite (/home/travis/build/sfdye/pr-merge/node_modules/jest-jasmine2/build/jasmine/Env.js:334:25)
      at Env.describe (/home/travis/build/sfdye/pr-merge/node_modules/jest-jasmine2/build/jasmine/Env.js:303:7)
      at describe (/home/travis/build/sfdye/pr-merge/node_modules/jest-jasmine2/build/jasmine/jasmine_light.js:97:157)
      at Object.<anonymous> (/home/travis/build/sfdye/pr-merge/__tests__/index.test.js:27:1)
      at Runtime._execModule (/home/travis/build/sfdye/pr-merge/node_modules/jest-runtime/build/index.js:520:13)
      at Runtime.requireModule (/home/travis/build/sfdye/pr-merge/node_modules/jest-runtime/build/index.js:332:14)
      at /home/travis/build/sfdye/pr-merge/node_modules/jest-jasmine2/build/index.js:101:13
      at Generator.next (<anonymous>)
      at step (/home/travis/build/sfdye/pr-merge/node_modules/jest-jasmine2/build/index.js:106:583)
      at /home/travis/build/sfdye/pr-merge/node_modules/jest-jasmine2/build/index.js:106:813
      at /home/travis/build/sfdye/pr-merge/node_modules/jest-jasmine2/build/index.js:106:494
      at jasmine2 (/home/travis/build/sfdye/pr-merge/node_modules/jest-jasmine2/build/index.js:106:68)
      at runTest (/home/travis/build/sfdye/pr-merge/node_modules/jest-runner/build/run_test.js:98:10)
      at /home/travis/build/sfdye/pr-merge/node_modules/jest-runner/build/index.js:71:68
      at Generator.next (<anonymous>)
      at step (/home/travis/build/sfdye/pr-merge/node_modules/jest-runner/build/index.js:23:376)
      at /home/travis/build/sfdye/pr-merge/node_modules/jest-runner/build/index.js:23:536
      at process._tickCallback (internal/process/next_tick.js:109:7)
  --
  event: {
    "event": "pull_request.closed",
    "installation": 35029
  }
Test Suites: 1 passed, 1 total
Tests:       2 passed, 2 total
Snapshots:   0 total
Time:        1.998s
Ran all test suites.
standard: Use JavaScript Standard Style (https://standardjs.com)
standard: Run `standard --fix` to automatically fix some problems.
  /home/travis/build/sfdye/pr-merge/__tests__/index.test.js:21:14: Missing space before function parentheses.
npm ERR! Test failed.  See above for more details.
```

--- a/.eslintrc
+++ b/.eslintrc
@@ -1,0 +1,38 @@
+{
+  "extends": "airbnb-base/legacy",
+  "parserOptions": {
+    "ecmaVersion": 2017,
+    "sourceType": "module"
+  },
+  "globals": {
+    "it": false,
+    "before": false,
+    "after": false,
+    "beforeEach": false,
+    "afterEach": false,
+    "describe": false,
+    "expect": false,
+    "jest": false,
+    "alert": false
+  },
+  "rules": {
+    "semi": [2, "never"],
+    "no-unused-expressions": 0,
+    "no-underscore-dangle": 0,
+    "max-len": [2, 150, 4, {"ignoreUrls": true}],
+    "new-cap": [2, { "capIsNewExceptions": ["Map", "List", "Stack", "Record", "StackNavigator", "DrawerNavigator", "TabNavigator"]}],
+    "func-names": 0,
+    "quotes": [2, "single"],
+    "no-use-before-define": [2, { "functions": false }],
+    "import/prefer-default-export": 0,
+    "global-require": 0,
+    "arrow-parens": ["error", "always"],
+    "arrow-body-style": 0,
+    "no-nested-ternary": 0,
+    "space-before-function-paren": ["error", {
+      "anonymous": "always",
+      "named": "always",
+      "asyncArrow": "always"
+    }]
+  }
+}

--- a/__tests__/fixtures/payload.json
+++ b/__tests__/fixtures/payload.json
@@ -1,0 +1,26 @@
+{
+  "event": "pull_request",
+  "payload": {
+    "action": "closed",
+    "pull_request": {
+      "number": 21,
+      "user": {
+        "login": "owner"
+      },
+      "title": "This is a pr",
+      "merged": true,
+      "head": {
+        "label": "This is a label"
+      }
+    },
+    "repository": {
+      "name": "testing-things",
+      "owner": {
+        "login": "owner"
+      }
+    },
+    "installation": {
+      "id": 35029
+    }
+  }
+}

--- a/__tests__/index.test.js
+++ b/__tests__/index.test.js
@@ -1,10 +1,69 @@
-// You can import your modules
-// const index = require('../index')
+const { createRobot } = require('probot')
+const sinon = require('sinon')
+const app = require('../index')
+const payload = require('./fixtures/payload')
 
-test('that we can run tests', () => {
-  // your real tests go here
-  expect(1 + 2 + 3).toBe(6)
+const initMock = (github) => {
+  const robot = createRobot()
+  robot.auth = sinon.stub().resolves(github)
+  app(robot)
+
+  return robot
+}
+
+const getGithubMock = () => ({
+  issues: {
+    createComment: sinon.stub()
+  }
 })
 
-// For more information about testing with Jest see:
-// https://facebook.github.io/jest/
+class NotFoundError extends Error {
+  constructor (code) {
+    super()
+    this.code = code
+  }
+}
+
+describe('app plugin test', () => {
+  describe('should resolve', () => {
+    let robot
+    let githubMock = getGithubMock()
+
+    beforeEach(() => {
+      robot = initMock(githubMock)
+      githubMock.issues.createComment.resolves()
+    })
+
+    it('should performs an action', async () => {
+      const expectedParams = {
+        number: 21,
+        owner: 'owner',
+        repo: 'testing-things',
+        body: 'Congratulations on the merge :tada: Don\'t forget to delete the branch `This is a label`.'
+      }
+
+      await robot.receive(payload)
+
+      expect(githubMock.issues.createComment.calledOnce).toEqual(true)
+      expect(githubMock.issues.createComment.args[0][0]).toEqual(expectedParams)
+    })
+  })
+
+  describe('should reject', () => {
+    let robot
+    let githubMock = getGithubMock()
+
+    beforeEach(() => {
+      githubMock.issues.createComment.rejects(new NotFoundError())
+      robot = initMock(githubMock)
+    })
+
+    it('should throw exception', async () => {
+      try {
+        await robot.receive(payload)
+      } catch (err) {
+        expect(err instanceof NotFoundError).toEqual(true)
+      }
+    })
+  })
+})

--- a/index.js
+++ b/index.js
@@ -1,7 +1,9 @@
 module.exports = (robot) => {
-  robot.on('pull_request.closed', async context => {
-    if (context.payload.pull_request.merged) {
-      const branch = context.payload.pull_request.head.label
+  robot.on('pull_request.closed', async (context) => {
+    const { merged, head } = context.payload.pull_request
+
+    if (merged) {
+      const branch = head.label
       const params = context.issue({
         body: `Congratulations on the merge :tada: Don't forget to delete the branch \`${branch}\`.`
       })

--- a/package.json
+++ b/package.json
@@ -10,9 +10,13 @@
     "test": "jest && standard"
   },
   "dependencies": {
-    "probot": "^5.0.0"
+    "probot": "^5.0.0",
+    "sinon": "^4.2.2"
   },
   "devDependencies": {
+    "eslint": "^4.17.0",
+    "eslint-config-airbnb-base": "^12.1.0",
+    "eslint-plugin-import": "^2.8.0",
     "jest": "^21.2.1",
     "smee-client": "^1.0.1",
     "standard": "^10.0.3"

